modules/perception/onboard/msg_serializer/msg_serializer.cc

```diff
@@ -184,17 +184,17 @@ bool MsgSerializer::ConvertObjectToPb(const base::ObjectPtr &object_ptr,
     }
   }
 
-  // record the best prediction trajectory
-  if (object_ptr->feature.get() &&
-      object_ptr->feature->predicted_trajectory_size() > 0) {
-    apollo::perception::DebugMessage *dmsg = pb_msg->mutable_msg();
-    apollo::perception::Trajectory *target_traj = dmsg->add_trajectory();
-    const apollo::prediction::Trajectory &src_traj =
-        object_ptr->feature->predicted_trajectory(0);
-    (*target_traj->mutable_trajectory_point()) = (src_traj.trajectory_point());
-    ADEBUG << "Inserting Trajectores in PB with point size "
-           << src_traj.trajectory_point_size();
-  }
+//  // record the best prediction trajectory
```

Comment:
Put a TODO to revisit it at a later time or delete the lines if they are no longer needed.